guides/common/modules/proc_creating-variables-for-convert2rhel-role.adoc

```diff
@@ -5,22 +5,27 @@ Assigning the Ansible role to a host also installs variables.
 You must create the following additional variables:
 [cols="20%,10%,15%",options="header"]
 |====
-| Name | Type | Default value to set
+| Name | Type | Default value
 | {project-context}_rhel_wait_for_syncs | boolean | false
 | {project-context}_manifest_path | string | path to manifest
 | {project-context}_organization | string | name of organization
 | {project-context}_password | string | your admin password
 | {project-context}_server_url | string | URL of {project-context} instance
 | {project-context}_username | string | admin
-| {project-context}_validate_certs |boolean | do ansible cert checks?
+| {project-context}_validate_certs |boolean | Enable certificate check in Ansible (default: true)
+| {project-context}_content_rhel_enable_rhel7 | boolean | true if converting to RHEL 7
```

Comment:
```suggestion
| {project-context}_content_rhel_enable_rhel7 | boolean | Set to `true` if converting to {RHEL} 7
```

Maybe you can also rework other descriptions if you deem necessary/an improvement.